common/src/main/java/com/wynntils/features/user/TranslationFeature.java

```diff
@@ -110,8 +110,8 @@ private String wrapCoding(String origCoded) {
     }
 
     private static class TranslatedNpcDialogEvent extends NpcDialogEvent {
-        public TranslatedNpcDialogEvent(String codedDialog) {
-            super(codedDialog);
+        public TranslatedNpcDialogEvent(String codedDialog, boolean autoProgressing) {
```

Comment:
Also, to be clear, I think the name "nonBlocking" (or perhaps even better if we switch it around and use "blocking" (which then will be true for the "traditional" NPC dialogues) should be propagated everywhere, into this event and where the event is read, etc.

Reply:
Should as in "I agree, I need to fix this", or "this should have been fixed"? Because if it's the latter, that does not seem to be the case, or there is something weird with what github shows.